tests/trigger_event_tests.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import mock
+
+from django.conf import settings
+from django.test import TestCase
+from django.test.utils import override_settings
+from django.contrib.auth.models import User
+
+from django_emarsys.event import trigger_automatic_event, trigger_manual_event
+from django_emarsys.models import NewEventInstance
+
+
+class TriggerEventTestCase(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.user = User.objects.create(username="test_user",
+                                       email="test.user@machtfit.de")
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.user.delete()
+
+    @override_settings()
+    @mock.patch("django_emarsys.api.get_events")
+    def test_trigger_event_with_unknown_name(self, mock_api_get_events):
+        mock_api_get_events.return_value = {}
+        settings.EMARSYS_EVENTS = {}
+        event = trigger_automatic_event("test event", "oliver.runge@machtfit.de")
```

Comment:
Not sure you want to expose this poor guy's email address unnecessarily to the relentlessly spamming internets :)